src/pseudopeople/noise_functions.py

```diff
@@ -392,3 +390,18 @@ def keyboard_corrupt(truth, corrupted_pr, addl_pr, rng):
 #     """
 #     # todo actually generate OCR errors
 #     return column
+
+
+####################
+# HELPER FUNCTIONS #
+####################
+
+
+def _coerce_dtype(new_values):
+    required_dtype = [c.dtype for c in schema_entities.COLUMNS if c.name == new_values.name][
+        0
+    ]
+    if new_values.dtype != required_dtype:
```

Comment:
It seems to work as-is (which I thought was odd tbh). I'll add .name though just to be safe

Reply:
@rmudambi Actually, checking that the name is correct won't work for categorical (for which the only differing ones are states because of sample data having US.